.github/workflows/flaky-tests-bypass.yaml

```diff
@@ -15,6 +15,9 @@ on:
   pull_request:
     paths-ignore:
       - '**.go'
+  merge_group:
```

Comment:
Does the merge_group should be also added in flaky-tests.yaml file ?

Reply:
Yeah, `merge_group` trigger should be added to both workflows with the same path filters as their respective `pull_request` trigger. Otherwise it will get stuck.